quizcon/templates/main/index.html

```diff
@@ -1,14 +1,55 @@
 {% extends 'base.html' %}
 
-{% block title %}Splash{% endblock %}
+{% block title %}Sign In | QuizCon{% endblock %}
+{% block topnavbar %}
+{% endblock %}
+{% block css %}
+<style>
```

Comment:
Just one nit, to avoid having to inline styles, you might want to use a template block to add classes to the body tag. You can use that class name to scope these styles to just the home page.